src/main/java/com/wolox/training/models/User.java

```diff
@@ -30,64 +38,31 @@
     @NotNull
     private LocalDate birthdate;
 
-    @ManyToMany(mappedBy = "users")
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(name = "books_users",
+            joinColumns = @JoinColumn(name = "user_id", referencedColumnName = "id"),
+            inverseJoinColumns = @JoinColumn(name = "book_id",
+                    referencedColumnName = "id"))
```

Comment:
Why are you removing the mappedBy and changing for a whole definition, even with the JoinTable data?
Only one side of the relationship should be the owner.